cmd/tests/cmd_run_test.go

```diff
@@ -42,7 +42,7 @@ func TestVersion(t *testing.T) {
 	cmd.ExecuteWithGlobalState(ts.GlobalState)
 
 	stdout := ts.Stdout.String()
-	assert.Contains(t, stdout, "k6 v"+consts.Version)
+	assert.Contains(t, stdout, "k6 version "+consts.Version)
```

Comment:
It prints
```
k6 version 0.47.0 (go1.21.1, linux/amd64)
```

That means it would be a breaking change for someone relying on it on automated tools. And, it doesn't match really the version we usually announce, check https://github.com/grafana/k6/releases/tag/v0.47.0 for an example.

Can you roll back it, please? I expect you're doing it because it is the default template for Cobra but if you use [SetVersionTemplate](https://pkg.go.dev/github.com/spf13/cobra#Command.SetVersionTemplate) then you should be able to change it.

Reply:
Hi @codebien !

Thank you for your feedback. I did change that and the corresponding test to match Cobra's default. 
I'll rollback that and change the default.